examples/object_detector/yolo/yolov8/custom_handler.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+from collections import defaultdict
+
+import torch
+from torchvision import transforms
+from ultralytics import YOLO
+
+from ts.torch_handler.object_detector import ObjectDetector
+from ts.utils.util import load_label_mapping
+
+logger = logging.getLogger(__name__)
+
+try:
+    import torch_xla.core.xla_model as xm
+
+    XLA_AVAILABLE = True
+except ImportError as error:
+    XLA_AVAILABLE = False
+
+
+class Yolov8Handler(ObjectDetector):
+    image_processing = transforms.Compose(
+        [transforms.Resize(640), transforms.CenterCrop(640), transforms.ToTensor()]
+    )
+
+    def __init__(self):
+        super(Yolov8Handler, self).__init__()
+
+    def initialize(self, context):
+        properties = context.system_properties
+        if torch.cuda.is_available() and properties.get("gpu_id") is not None:
+            self.map_location = "cuda"
+            self.device = torch.device(
+                self.map_location + ":" + str(properties.get("gpu_id"))
+            )
+        elif XLA_AVAILABLE:
+            self.device = xm.xla_device()
+        else:
+            self.map_location = "cpu"
+            self.device = torch.device(self.map_location)
+
+        self.manifest = context.manifest
+
+        model_dir = properties.get("model_dir")
+        self.model_pt_path = None
+        if "serializedFile" in self.manifest["model"]:
+            serialized_file = self.manifest["model"]["serializedFile"]
+            self.model_pt_path = os.path.join(model_dir, serialized_file)
+
+        self.model = self._load_torchscript_model(self.model_pt_path)
+
+        logger.debug("Model file %s loaded successfully", self.model_pt_path)
+
+        # Load class mapping for classifiers
+        mapping_file_path = os.path.join(model_dir, "index_to_name.json")
+        self.mapping = load_label_mapping(mapping_file_path)
```

Comment:
interesting same mapping applies?

Reply:
Cleaned it up. Copy paste artifact.
But, I've been thinking about how we should have most of the default `initialize` in `try` `except` blocks. This way we don't have to copy paste code. Thoughts?